### ISSUE TITLE
Bug while analysing my game

### ISSUE THREAD

**Issue by dotslashbit** [lichess-org/chessground#239]
After I played my game, I clicked on the analyze game button, and in this analysis tab, in the bottom right corner there is a button through which I can get to the last position of the game and there is a button through which I can get to the starting position of the game.
If you'll right click on any of them you won't directly jump to starting or the ending position of the game but it'll just scroll through all the moves very fast.

**Comment by benediktwerner** [comment]
Can't reproduce. Which browser and OS are you using?

**Comment by benediktwerner** [comment]
Actually, this isn't the right repo for this anyway. I assume you're talking about a Lichess issue?

**Comment by dotslashbit** [comment]
Yes, I'm talking about a lichess issue. Can you tell me which repo to go to for opening this issue?
